bokehjs/src/coffee/common/plot.coffee

```diff
@@ -122,6 +123,11 @@ class PlotView extends ContinuumView
 
     @update_dataranges()
 
+    if @mget('responsive')
+      throttled_resize = _.throttle(@resize, 100)
+      $(window).on("resize", throttled_resize)
+      $(@resize)
```

Comment:
Technically, the container could change size also without the window changing size. But ok for now, I think we can just hook into Phosphor resize messages/signals at some point.